projects/agent-test-harness.md

```diff
@@ -0,0 +1,102 @@
+# Agent Test Harness
+
+## Project Name: To Be Determined
+
+In proposing to bring the current [Aries Agent Test Harness](https://aries-interop.info) to OWF, we suggest merging into it the [Mobile Agent Test Harness](https://github.com/openwallet-foundation/mobile-wallet-test-harness), currently (but not supposed to be) part of the OWF”s Bifold Wallet project. The Mobile Agent Test Harness uses Aries Agent Test Harness in some of its test cases. We also propose including in this testing project the [Aries Akrida](https://github.com/hyperledger/aries-akrida) repository, an agent load testing tool for deployments of ACA-Py and compatible agents. Akrida’s engine is based on the open source [Locust](https://locust.io/) tool, and instances (single or clustered) of OWF’s [Credo-TS](https://github.com/openwallet-foundation/credo-ts). Together, the three components provide rich testing of Agents and Wallets for compatibility, interoperability, and scalability.
+
+## Preferred Maturity Level
+
+We propose that the project be at the “Growth” level.  All three of the components have been used extensively in the past and are well documented. Many groups have used the tools in testing their own instances of the various support agents and wallets.  As we move the project to OWF and start up the Wallet Interoperability Special Interest Group (link to PR to be added) these tools will be useful in enabling testing across frameworks and deployments built on those deployments. We look to expand their use in both the scope of the testing they enable, and their use by the community.
```

Comment:
Can you comment on how the project meets each of the "Growth" acceptance criteria outlined in the [project lifecycle](https://tac.openwallet.foundation/governance/project-lifecycle/)?

Reply:
Added how the Growth stage requirements are met to the proposal.